bin/run

```diff
@@ -1,4 +1,20 @@
 #!/usr/bin/env node
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+// support -h in addition to --help everywhere
+process.argv = process.argv.map(v => v === "-h" ? "--help" : v);
```

Comment:
Ima do this everywhere

Reply:
It works on the top level of oclif clis, for example `aio -h` because that's handled [here in the Main command](https://github.com/oclif/command/blob/136e7d4f35c7ac1087e6af920dcadf21bc73530f/src/main.ts#L30).

However, for commands, e.g. `aio app deploy -h` it doesn't work because it's a [different place in Command](https://github.com/oclif/command/blob/baf3576ba23a4901c8d99f094f1267cd4ff30fd7/src/command.ts#L204) that only accepts `--help`.

Be aware that blindly doing the above might interfere with commands that use their command args differently. Say `aio something do -- call-other-tool -h` where you maybe don't want `-h` to be modified.